include/render/DepthTexture.h

```diff
@@ -30,39 +30,64 @@
 class DepthTexture
 {
 public:
-  DepthTexture();
-  ~DepthTexture();
+	DepthTexture(int window_width, int window_height, int framebuffer_width, int framebuffer_height);
+	~DepthTexture();
 
-  void copyDepthbuffer();
+	void copyDepthbuffer();
 
-  unsigned int depth_texture()
-  {
-    return m_depth_texture;
-  };
+	unsigned int depth_texture()
+	{
+		return m_depth_texture;
+	};
 
-  const unsigned &width() const
-  {
-    return m_width;
-  }
+	unsigned int width() const
+	{
+		return m_width;
+	}
 
-  const unsigned &height() const
-  {
-    return m_height;
-  }
+	unsigned int height() const
+	{
+		return m_height;
+	}
 
-  static void getJetColor(double value, double min_val, double max_val, float &r, float &g, float &b, float &a);
+
+	unsigned int frame_buffer_width() const
+	{
+		return m_framebuffer_width;
+	}
```

Comment:
What's the tab size your formatter is using? It seems so bigger but it may just be because I'm seeing it on GitHub